test/unit/puppetlabs/services/jruby/jruby_puppet_testutils.clj

```diff
@@ -17,6 +17,8 @@
 
 (def ruby-load-path ["./ruby/puppet/lib" "./ruby/facter/lib" "./ruby/hiera/lib"])
 (def gem-home "./target/jruby-gem-home")
+(def environment-vars {"FOO" "for_jruby"
+                       "BAR" "also_for_jruby"})
```

Comment:
I think that the keys should be keywords since that's what's going to be passed in by the HOCON config.